README.md

```diff
@@ -1 +1,12 @@
-# blue-harvest
\ No newline at end of file
+# blue-harvest
+
+### It seems that blue-harvest is not very mature yet
+* We can either modify this forked code to suit our needs (which we are doing in the short term)
```

Comment:
We included screenshot testing and action helpers in the same repo for the purposes of the demo. However, it probably makes more sense in the long run for them to be separate projects. If you wanted to start a new project just focused on screenshot testing, starting with ideas from blue-harvest, I think that'd be a great idea.

Reply:
@heathkit that is not a bad idea.  Let us think about it, and look into whether there is one already that we can use but for the moment we can update this repo.